### PR TITLE
fix: consistent border rad; click-to-copy

### PR DESCRIPTION
**Pull request by JFrankfurt**
image

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

**Review by kirkas — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for kirkas @ 2024-08-02 00:59:42 UTC
User failed mfa authentication, see [go/mfa-help](http://go/mfa-help)

File: apps/web/src/components/Basenames/RegistrationFlow.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ export function RegistrationFlow() {
 
   return (
     <>
-      {isDevelopment && <RegistrationStateSwitcherDynamic />}
+      {false && isDevelopment && <RegistrationStateSwitcherDynamic />}
```
Comment (zencephalon) [inline apps/web/src/components/Basenames/RegistrationFlow.tsx:118]: Deliberate?

Comment (JFrankfurt) [reply apps/web/src/components/Basenames/RegistrationFlow.tsx:118]: yeah, imo this should be opt-in if you need it, and was messing with the menu I was working on here

**Review by kirkas — APPROVED** [review]

**Pull request merged**